content/sensu-go/5.9/api/authproviders.md

```diff
@@ -9,14 +9,14 @@ menu:
     parent: api
 ---
 
-- [The `authproviders` API endpoints](#the-authproviders-api-endpoints) (enterprise-only)
+- [The `authproviders` API endpoints](#the-authproviders-api-endpoints) (licensed tier)
   - [`/authproviders` (GET)](#authproviders-get)
-- [The `authproviders/:name` API endpoints](#the-providersname-api-endpoints) (enterprise-only)
+- [The `authproviders/:name` API endpoints](#the-providersname-api-endpoints) (licensed tier)
   - [`authproviders/:name` (GET)](#providersname-get)
   - [`authproviders/:name` (PUT)](#providersname-put)
   - [`authproviders/:name` (DELETE)](#authprovidersname-delete)
 
-**ENTERPRISE ONLY**: Authentication providers in Sensu Go require an enterprise license. To activate your enterprise license, see the [getting started guide][2].
+**LICENSED TIER**: Unlock authentication providers in Sensu Go with a Sensu license. To activate your license, see the [getting started guide][2].
```

Comment:
"Unlock" is 👌

Reply:
This is especially good, directionally speaking. It's presented in a more active voice; "here's how to do X" rather than "in order to do X, you're going to need to do something <$$$>". It is presented in a much more frictionless manner. Very well done!

Reply:
/cc @bananabele 👆 (as I'm sure you'll agree). 😄